notebooks/data_generator.py

```
@@ -0,0 +1,458 @@
+            self.generate_columns()
+        else:
+            print("Original dataset is being used. Cant create fraud entries generate seperate data and then concatenate")
+
+    def concat_data(self,new_data):
+        try:
+
+            merged_data = pd.concat([self.df, new_data], axis=0, ignore_index=True)
+
+            # Shuffle the merged dataset
+            self.df = merged_data.sample(frac=1).reset_index(drop=True)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)